styles/widgets/generic/toolbar.generic.less

```diff
@@ -180,3 +189,22 @@
 .dx-toolbar .dx-tab {
     padding: 4px;
 }
+
+.dx-toolbar-button {
+    .dx-menu {
+        .dx-menu-item {
+            border: 1px solid transparent;
+            border-radius: 4px;
```

Comment:
Should we use @base-border-radius here?

Reply:
A menu item in a toolbar should look like a button so I use button's parameters here

Reply:
@button-border-radius is fine too.